test/bigtest/tests/match-profiles/match-profile-details-test.js

```diff
@@ -200,11 +200,8 @@ describe('Match Profile View', () => {
               expect(matchProfileDetails.matchCriteria.incomingRecordSections.children(1).label).to.be.equal('Use a qualifier');
             });
 
-            it('has empty QualifierType field value', () => {
-              expect(matchProfileDetails.matchCriteria.fieldQualifierType.value.text).to.be.equal('-');
-            });
-            it('has empty QualifierValue field value', () => {
-              expect(matchProfileDetails.matchCriteria.fieldQualifierValue.value.text).to.be.equal('-');
```

Comment:
I think it will be good to have a covered with tests case when the `Use a qualifier` section has one filled and one empty field.

Reply:
will be added in the future